DocumentService/Word/WordDocumentGenerator.cs

```diff
@@ -83,17 +87,28 @@ public static void GenerateDocumentByTemplate(string templateFilePath, DocumentD
                 // Write the document to output file path and close the document
                 WriteDocument(document, outputFilePath);
                 document.Close();
+
+                /*
+                 * Image Replacement is done after writing the document here,
+                 * because for Text Replacement, NPOI package is being used
+                 * and for Image Replacement, OpeXML package is used.
+                 * Since both the packages have different execution method, so they are handled separately
+                 */
+                // Replace all the image placeholders in the output file
+                ReplaceImagePlaceholders(outputFilePath, outputFilePath, imagePlaceholders);
             }
             catch (Exception ex)
             {
                 Console.WriteLine("Error: " + ex.Message);
```

Comment:
Can you remove `Console.WriteLine`?
I don't see any value in writing the output on the console.

Reply:
Updated the catch block - removed console log statement